manual/C/ch_Finance-Quote.xml

```diff
@@ -58,9 +58,12 @@ Translators:
 
     <para>To check if &app-perl; is already installed on your system, call the command
       <userinput>perl -v</userinput> in the terminal.
+    <informalexample><?dbfo pgwide="1"?>
```

Comment:
For test purposes this is OK, but before release it should go in the XSLT customization.

Perhaps this should also be used on large images.

Reply:
XSLT is a mystery to me.

Reply:
aber du hast doch schon mal an `COMMAND  ${XSLTPROC}` einen `--stringparam` angefügt, oder?
Sauberer wäre natürlich http://www.sagehill.net/docbookxsl/ParametersInFile.html, aber damit würde ich gerne warten, bis die `ghelp:`-Geschichte gelöst ist:
https://lists.gnucash.org/pipermail/gnucash-user/2022-September/102726.html

Reply:
Please create a bug with the content of the discussion and add a referencing Fixme: comment here.

Reply:
Not in this PR, before line 61 `<!-- Fixme: https://bugs.gnucash.org/show_bug.cgi?id=798618 -->`